crates/gitql-ast/src/function.rs

```diff
@@ -474,6 +482,12 @@ fn date_current_timestamp(_inputs: Vec<Value>) -> Value {
     return Value::DateTime(time_stamp);
 }
 
+fn date_now(_inputs: Vec<Value>) -> Value {
```

Comment:
No need to create a new function for now you can just use `date_current_timestamp` because current_timestamp and now are the same but now is works only on MYSQL